### DISCUSSION TITLE
Throttle By display command?

### DISCUSSION THREAD

**Discussion by Nobeernogman** [danielnilsson9/bbs-fw#42] [Q&A]
1) The original fw/config tool also have a option "By display command" I always use this function, i love it.
https://i.imgur.com/l7ldORk.png
I can't find it in the custom fw tool. Is is not (yet?) available, if not, i hope it wil come soon. 

2) Is it also possible to have PAS/Throttle and cruise control at the same time?

Kind regards

**Comment by danielnilsson9** [comment]
1. Throttle config is set by assist level and can be configured differently per assist level. So in this firmware throttle power is always by "displays command". If you want to always have full throttle power available to you configure throttle (%) to 100 for all assist levels.

2. Not on the same assist level since the throttle is used for engaging cruise. But you can have different types on different assist levels.

https://github.com/danielnilsson9/bbshd-fw/wiki/Configuration-Tool
